tests/test_validation.py

```diff
@@ -199,6 +201,34 @@ def validate(args, limit_to=None):
         self.assertTrue('could not be interpreted as a number'
                         in validation_errors[0][1])
 
+    def test_timeout_succeed(self):
+        from natcap.invest import validation
+
+        # both args and the kwarg should be passed to the function
+        def func(arg1, arg2, kwarg=None):
+            self.assertEqual(kwarg, 'kwarg')
+            time.sleep(1)
+
+        # this will raise an error if the timeout is exceeded
+        # timeout defaults to 5 seconds so this should pass
+        validation.timeout(func, 'arg1', 'arg2', kwarg='kwarg')
+
+    def test_timeout_fail(self):
+        from natcap.invest import validation
+
+        # both args and the kwarg should be passed to the function
+        def func(arg):
+            time.sleep(10)
+
+        # this will return a warning if the timeout is exceeded
+        # timeout defaults to 5 seconds so this should fail
+        with warnings.catch_warnings(record=True) as ws:
+            # cause all warnings to always be triggered
+            warnings.simplefilter("always")
+            validation.timeout(func, 'arg')
+            self.assertTrue(len(ws) == 1)
+            self.assertTrue('timed out' in str(ws[0].message))
```

Comment:
Do you think it would be reasonable to trim down the sleep time here?  10s isn't a terribly long time to wait in the scope of the rest of the test suite, but it also doesn't seem like that long of a wait is required to test the timeout function.

Reply:
Yes definitely, 6 seconds should be fine